scripts/lib.sh

```diff
@@ -81,7 +81,7 @@ start-docker() {
 
 upgrade-pip() {
     # pip versions before 20.1 do not have `pip cache` as a command which is necessary for the CI
-    pip install --no-cache-dir --upgrade "pip>=20.1"
+    pip3 install --no-cache-dir --upgrade "pip==21.1.2"
```

Comment:
Let's not use `pip3` unless for some reason there was a problem with `pip` which there shouldn't be.

Reply:
I can get by with that. It's only a problem if that function is called within the context of a Python 2 environment (which only happens when I'm playing with things locally).

Reply:
Well, it seems I had it because calling `make upgrade-pip` early as part of the CI gives me Python 2 by default.

I worked around it by switching the call `pip cache dir` to `pip3 cache dir`. No need to touch `upgrade-pip`.